### DISCUSSION TITLE
Directus_users form read / write access

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16007] [Archive]
How to solve the following use case:
>Apart the `administrator`, the app is ruled by `managers` and used by `regular users`.
The `managers` are non technical people with no `Admin access`.
They should be able to edit and correct some fields (not all) of `regular users` and all fields of their own profile.

To get this, in the present Directus version, we allow

- `regular users` to edit any field of their own profile (except Access and Role)
- `managers` to edit some fields (not all) of any `regular users`.

Question: How do we allow the `managers` to edit all the fields of their own profile ( apart giving them full edit access on `regular users`)?

**Comment by rijkvanzanten** [comment]
Great question. I don't think that's currently possible in the rbac system. The fields that are allowed to be edited aren't "conditional", so you can't say "You can edit fields A, B, C when value A = 'x', or fields C, D, E when B = 'y'". Also wondering if that's something we could do without a big performance hit, as we'd have to read the item and run the validation on a per-item basis first, rather than validating against the incoming payload 🤔

**Reply by connorwinston** [reply]
This could be a really powerful addition as it would basically allow you to lock down on the api what you can do with the field conditional read only/hidden on display conditional.
